Jenkinsfile

```diff
@@ -70,6 +70,12 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
             // Default config
             sh "cp ${params.CONFIG_FILE?.trim()} .config"
 
+            // Package server config
+            if (params.VERSION) {
+                echo 'Updating server location for package downloads'
+                sh "sed -i '/^CONFIG_VERSION_REPO/s/latest/ci40-${params.VERSION}/' .config"
```

Comment:
FYI, I was trying the sed command below which will be consistent with other sed commands in this Jenkinsfile, but it gives an error, so went the above format

    $ sed -i '|^CONFIG_VERSION_REPO|s|latest|ci40-v1.0.1|' pistachio.config 
    sed: -e expression #1, char 1: unknown command: `|'